config/locales/notices/en.yml

```diff
@@ -16,25 +16,24 @@ en:
       resend_email_success: We have resent your password reset email.
     password_changed: You changed your password.
     resend_confirmation_email:
-      success: We have resent your confirmation email.
+      success: We resent your confirmation email.
```

Comment:
No, @jessieay, first glance, I read it the same way, which was why I suggested perhaps writing, `We sent you another confirmation email.`